pkg/terminal/proxy.go

```diff
@@ -0,0 +1,173 @@
+package terminal
+
+import (
+	"crypto/tls"
+	"errors"
+	"github.com/openshift/console/pkg/auth"
+	"github.com/openshift/console/pkg/proxy"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/rest"
+	"net/http"
```

Comment:
we have a convention for imports, builtin first, then k8s, then openshift, then local.